canvas_data_warehouse/urls.py

```diff
@@ -0,0 +1,7 @@
+'''Register URL routes in auth module.'''
+from rest_framework import routers
+from . import views
+
+router = routers.SimpleRouter()
+router.register(r'data-graph', views.CanvasDataViewSet, base_name='data-graph')
```

Comment:
The viewset `CanvasDataViewSet` is for `canvas data`,  change registering url path and base name to be consistent.